src/MultiSelect/MultiSelect.tsx

```diff
@@ -260,7 +260,11 @@ const MultiSelect: React.FC<Props> = ({
             )}
           />
         </div>
-        <button className={cssClass.CARET_BUTTON} {...getToggleButtonProps()}>
+        <button
+          aria-label="toggle dropdown options"
```

Comment:
My only issue with this is that it's in English.

if you can confirm that nothing else in `clever-components` is internationalized, then this change is fine and we deal w/ i18n with `dew3y`.  

The alternative approach is that the user of this component passes in the aria-label value, which is awkward b/c it means the client has too much knowledge about the implementation of this component.

The right way is to actually support i18n with the component itself.

so long winded way of saying this is not an issue, if indeed nothing is i18n'ized in `clever-component`

Reply:
@zohnnyclever [here is](https://github.com/Clever/components/blob/master/src/MessagingBubble/NormalMessagingBubble.tsx#L42-L50) the only mention of `i18n` in the components repo. I can take this approach if it makes the most sense, but no other component seems to have taken these steps.

Reply:
oh, i remember doing that now. 

Yeah, i think you can do the same thing by adding a prominent comment about it. it's unfortunate b/c i can't think of a solution to this except for having `dew3y` support i18n and us migrating this component over to `dew3y`.